Dockerfile

```diff
@@ -0,0 +1,58 @@
+# Builder image
+FROM centos:7 as builder
+
+# Install EPEL repository for cmake3
+RUN yum -y install epel-release &&\
+	yum -y clean all
+
+# Install build tools and libraries
+RUN yum -y install gcc gcc-c++ make cmake3 \
+        glibc-devel libstdc++-devel libjpeg-turbo-devel libpng-devel zlib-devel \
+        glibc-devel.i686 libstdc++-devel.i686 libjpeg-turbo-devel.i686 libpng-devel.i686 zlib-devel.i686 &&\
+	yum -y clean all
+
+# Copy sources
+COPY . /usr/src/openjk
+
+# Build i386 arch
+RUN mkdir /usr/src/openjk/build.i386 &&\
+	cd /usr/src/openjk/build.i386 &&\
+	cmake3 -DCMAKE_TOOLCHAIN_FILE=CMakeModules/Toolchains/linux-i686.cmake \
```

Comment:
Maybe pass [CMAKE_INSTALL_PREFIX](https://cmake.org/cmake/help/v3.0/variable/CMAKE_INSTALL_PREFIX.html) and run `make install`?
This allows the server image to only copy the install tree.